R/utils.R

```diff
@@ -12,7 +12,6 @@ vnapply <- function(x, fun, ...) {
   vapply(x, fun, numeric(1), ...)
 }
 
-
```

Comment:
Following pep8 style I usually leave 2 lines between functions; it's never that consistent...